agent/api/task/task.go

```diff
@@ -1069,6 +1074,9 @@ func (task *Task) collectFirelensLogOptions(containerToLogOptions map[string]map
 				containerToLogOptions[container.Name] = make(map[string]string)
 			}
 			for k, v := range hostConfig.LogConfig.Config {
+				if k == firelensLogDriverBufferLimitOption {
+					LogDriverBufferLimit = v
+				}
```

Comment:
So this code gets the buffer limit from the user input- but how does it get added to the fluentd docker log driver configuration? I don't see that code in this PR....